hummingbot/core/utils/async_utils.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
+import logging
+
+
+def asyncio_ensure_future(*args, **kwargs):
+    try:
+        return asyncio.ensure_future(*args, **kwargs)
```

Comment:
This is not effective.

The `ensure_future()` call itself is not the problem - it won't cause any exceptions. It's the coroutine that gets scheduled that may have silent errors.

If you look at the definition of `ensure_future()` from Python's documentations, the first argument is the coroutine or await-able object. What you need to do is to wrap that thing in another async function.

e.g.

```
  async def safe_wrapper(c):
    try:
      return await c
    except asyncio.CancelledError:
      raise
    except Exception:
      # do some error log here

  # assume coro is defined here
  return asyncio.ensure_future(safe_wrapper(coro))
```

Reply:
Ok fixed.